chcemvediet/apps/inforequests/constants.py

```diff
@@ -0,0 +1 @@
+INFOREQUESTS_PER_PAGE = 50
```

Comment:
Kolko infoziadosti budeme zobrazovat na jednej stranke?

Reply:
@viliambalaz 50 je nateraz dobra konstanta. Mozno to v buducnosti zmenime.